public/stylesheets/game.sass

```diff
@@ -0,0 +1,24 @@
+$hand-border: 3px dashed white
+
+.container
+  display: table
+  border-collapse: collapse
+  height: 100%
+  width: 100%
+
+.row
+  display: table-row
+
+.hand
+  display: table-cell
+  height: 150px
+
+.trick
+  height: 100%
+  width: 100%
+
+.theirs
+  border-bottom: $hand-border
+
+.mine
+  border-top: $hand-border
```

Comment:
Need to move this into the plugin directory (and wire up the Sass processing appropriately).